src/irr/asset/CImageLoaderOpenEXR.cpp

```diff
@@ -213,6 +214,13 @@ namespace irr
 					const auto mapOfChannels = data.second;
 					PerImageData perImageData;
 
+					core::smart_refctd_dynamic_array<IImageMetadata::ImageInputSemantic> imageInputsMetadata = core::make_refctd_dynamic_array<decltype(imageInputsMetadata)>(1);
+					auto input = imageInputsMetadata->begin();
+					input->imageName = suffixOfChannels;
+					input->colorSpace = ECS_SRGB_NONLINEAR_KHR;
```

Comment:
AFAIK NONLINEAR sRGB will only be the case for non-floating point image format (image depth 8 bit or less)

float16 or higher will be LINEAR sRGB without an EOTF (linear EOTF)